package.json

```diff
@@ -13,7 +13,7 @@
     "official"
   ],
   "dependencies": {
-    "mongodb-core": "mongodb-js/mongodb-core#3.0.0"
+    "mongodb-core": "mongodb-js/mongodb-core#causally-consistent-reads"
```

Comment:
this will be updated when y'all approve https://github.com/mongodb-js/mongodb-core/pull/238